BringgSDK.js

```diff
@@ -815,6 +815,25 @@ var BringgSDK = (function () {
     }
   };
 
+
+  //==============================================
+  // Lazy Masked Phone Number
+  //==============================================
+  module.getDriverPhone = function (uuid, cb) {
+    if (!uuid) {
+      cb({status: 'error', message: 'No shared_uuid provided'});
+      return;
+    }
+    $.get(getRealTimeEndPoint() + '/shared/' + uuid + '/phone_number', {})
+      .success(function (result) {
+        cb(result);
+      }).fail(function (error) {
+        console.log(111)
+        console.log(error)
+        cb(JSON.parse(error.responseText));
```

Comment:
to follow the same api we provide in the other actions (note, tip, etc) we also put a success: false there